docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: '3'
+services:
+    chat-linker:
+      build: .
+      volumes:
+        - ./app-config.json:/usr/app/app-config.json:ro
```

Comment:
Are you sure it'll be working properly? You're copying `./app-config.json` already when doing `COPY . /usr/app`

I don't know if the volume config will be in effect in such case or not.

Reply:
the volume makes sure, that if you change the configuration... it will use the current values, not the ones when your built the docker image!

Reply:
Alright, if it works, I'll merge. Thanks for your contribution!